cypress/integration/app/email.spec.ts

```diff
@@ -0,0 +1,10 @@
+describe("Fleet Email API", () => {
+  it("returns JSON data for test@fleetdm.com", () => {
+    cy.request(
+      "http://localhost:8025/api/v2/search?kind=to&query=test@fleetdm.com"
+    )
+      .its("headers")
+      .its("content-type")
+      .should("include", "application/json");
+  });
+});
```

Comment:
I think we can remove this. We don't really need to besting Mailhog's API on its own.

Reply:
Ok, deleting this entire file :)